codecov.yml

```diff
@@ -75,21 +75,28 @@ flag_management:
 
 # Read more here: https://docs.codecov.com/docs/pull-request-comments
 comment:
+  #### FOR TESTING ####
+  # If you want to test in your PR commenting configurations changes,
+  # comment out this line and uncomment the next line in the next section
+  # require_bundle_changes: false
+  ######################
+  # Post comment if there are changes in bundle size increases greater than value specified
+  require_bundle_changes: 'bundle_increase'
+  ######################
   # This is the addition of carry forward builds and fresh builds, thus, it's the addition
   # of the FE and BE builds
   after_n_builds: 22
-  # Show only the git diff / patch coverage, and exclude any comparison between base and head:
-  layout: 'condensed_header, condensed_files, condensed_footer'
+  # What to render in the comment
+  layout: 'diff, condensed_files'
+  # Do not show the project coverage in the comment
   hide_project_coverage: true
+  show_carryforward_flags: true
   # Update, if exists. Otherwise post new. Skip if deleted.
   behavior: once
-  # Comments will only post when coverage changes. Furthermore, if a comment
-  # already exists, and a newer commit results in no coverage change for the
-  # entire pull, the comment will be deleted.
-  require_changes: true
+  # Only post if coverage drops OR there are uncovered lines of code
+  require_changes: 'coverage_drop OR uncovered_patch'
```

Comment:
Explained [here](https://docs.codecov.com/docs/pull-request-comments#requiring-changes).

> true - Only post comment if there are changes in coverage (positive or negative)

We don't care as much if the coverage increases.
> # Only post if coverage drops OR there are uncovered lines of code
> comment:
>    require_changes: "coverage_drop OR uncovered_patch"